packages/shared/src/browser.ts

```diff
@@ -52,8 +78,8 @@ export function userAgentIsRobot(userAgent: string): boolean {
  * @returns {boolean}
  */
 export function isValidBrowser(): boolean {
-  const navigator = window?.navigator;
-  if (!inBrowser() || !navigator) {
+  const navigator = browser.window.navigator;
```

Comment:
This changes the behavior - instead of returning `false` we're now throwing. Is this expected?